From 6e79c3ac4f1e6aeb7781813b5f65896b3c2b34d0 Mon Sep 17 00:00:00 2001
From: Alex Doyle <adoyle@nvidia.com>
Date: Fri, 18 Dec 2020 08:19:22 -0800
Subject: [PATCH] Patch the search path to search one directory up, so that
 files looking for <efivars/efivars.h> search  ${TOPDIR}/src/include and find 
 ${TOPDIR}/src/include/efivars/efivars.h ...instead of looking for: 
 ${TOPDIR}/src/include/efivars/efivars/efivars.h

---
 Make.defaults | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Make.defaults b/Make.defaults
index cf91cd1..7985d0d 100644
--- a/Make.defaults
+++ b/Make.defaults
@@ -30,8 +30,9 @@ ABIDW	:= abidw
 ABIDIFF := abidiff
 
 PKGS	=
-
+# ONIE - search src/include to NOT use host's header files.
 host_cflags := $(HOST_CFLAGS) -I${TOPDIR}/src/include/efivar/ \
+			-I${TOPDIR}/src/include/ \
 	$(if $(findstring clang,$(HOST_CC)),$(clang_cflags),) \
 	$(if $(findstring gcc,$(HOST_CC)),$(gcc_cflags),)
 
-- 
2.20.1

